tls/s2n_connection.c

```diff
@@ -359,6 +417,20 @@ int s2n_connection_set_send_cb(struct s2n_connection *conn, s2n_send_fn send)
     return 0;
 }
 
+int s2n_connection_set_cert_auth_type(struct s2n_connection *conn, s2n_cert_auth_type cert_auth_type)
```

Comment:
Could you add something in the documentation for this function that says that the default for the client is "REQUIRED" and the default for the server is "NONE"?